go.mod

```diff
@@ -66,6 +66,7 @@ require (
 	github.com/golang/protobuf v1.5.2 // indirect
 	github.com/golang/snappy v0.0.4 // indirect
 	github.com/gomodule/redigo v2.0.0+incompatible // indirect
+	github.com/google/go-github/v41 v41.0.1-0.20211227215900-a899e0fadbec
```

Comment:
This is fetching a specific commit of the [Go GitHub packager](https://github.com/google/go-github) off of version 41. It seems like they are due for a new release any time now. I imagine it would be best to wait for that before merging this(?). This is directly related to the closing of [this issue](https://github.com/google/go-github/issues/2232) I opened a bit ago.